README.md

```diff
@@ -44,6 +44,7 @@ raw:           false,  # Do not recursively format instance variables.
 sort_keys:     false,  # Do not sort hash keys.
 sort_vars:     true,   # Sort instance variables.
 limit:         false,  # Limit arrays & hashes. Accepts bool or int.
+depth:         nil,    # Limit number of nested objects the multiline can go deep.
```

Comment:
Rather than adding a new configuration, what do you think of allowing `multiline` to be an integer? We could support both true/false and 0+ values